examples/lighting-app/mbed/mbed_app.json

```diff
@@ -40,11 +40,13 @@
                 "NL_ASSERT_LOG=NL_ASSERT_LOG_DEFAULT",
                 "NL_ASSERT_EXPECT_FLAGS=NL_ASSERT_FLAG_LOG"
             ],
+            "target.components_add": [
+                "capsense"
+            ],
             "led-active-state": 0,
             "system-state-led": "LED1",
             "lighting-state-led": "P9_6",
-            "lighting-button": "USER_BUTTON",
-            "function-button": "P9_7"
```

Comment:
If we remove "function-button" here, how does "MBED_CONF_APP_FUNCTION_BUTTON" definition is resolved in AppTask.cpp? Additionally, I don't like the idea where` "function-button"` is still available for disco target (even if we assigned "NC" to it) but we removed it from cypress board.